01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
+    
+    $('.hidden').fadeIn(5000, function() {
```

Comment:
do you refer to leave a white space between the function keyword and the parentheses?